javascript/ql/lib/semmle/javascript/Arrays.qll

```diff
@@ -492,7 +492,20 @@ private module ArrayLibraries {
       exists(DataFlow::MethodCallNode call |
         call.getMethodName() = ["findLast", "find", "findLastIndex"] and
         prop = arrayLikeElement() and
-        obj = call.getReceiver() and
+        obj = call.getReceiver().getALocalSource() and
```

Comment:
```suggestion
        obj = call.getReceiver() and
```
In the `storeStep` you definitely need `.getALocalSource()`, but I'm not sure you need it here? 
See if this suggestion still passes the tests.

Reply:
64c45debdb4c1326f933d100eacac71c7898792d fixes.